components/elm/bld/namelist_files/namelist_defaults.xml

```diff
@@ -277,10 +277,14 @@ lnd/clm2/surfdata_map/surfdata_ne30pg2_simyr2010_c210402.nc</fsurdat>
 lnd/clm2/surfdata_map/surfdata_ne120np4_simyr2010_c20181023.nc</fsurdat>
 <fsurdat hgrid="ne1024np4.pg2"   sim_year="2010" use_crop=".false." >
 lnd/clm2/surfdata_map/surfdata_ne1024pg2_simyr2010_c211021.nc</fsurdat>
+<fsurdat hgrid="ne1024np4"   sim_year="2010" use_crop=".false." >
+lnd/clm2/surfdata_map/surfdata_ne1024np4_simyr2010_c220523.nc</fsurdat>
 
 <!-- for present day simulations - year 2000 -->
 <fsurdat hgrid="360x720cru"   sim_year="2000" use_crop=".false." >
 lnd/clm2/surfdata_map/surfdata_360x720cru_simyr2000_c180216.nc</fsurdat>
+<fsurdat hgrid="360x720cru"   sim_year="2010" use_crop=".false." >
+lnd/clm2/surfdata_map/surfdata_360x720cru_simyr2000_c180216.nc</fsurdat>
```

Comment:
Is this just a placeholder for now? It is ok to run with this file for 2010 compset. A surfdata_map for simyr2010 is expected to have noticeable differences in PCT_NAT_PFT, and round-off level (1e-7) differences in PCT_NATVEG, PCT_LAKE, PCT_GLACIER, PCT_URBAN. All other fields would be identical.